modules/foreman_setting.py

```diff
@@ -81,14 +81,17 @@
     name: "http_proxy"
 '''
 
-RETURN = ''' # '''
+RETURN = '''
+foreman_setting:
+  description: Created / Updated state of the setting
```

Comment:
Is this how the API returns it or a formatted version of the API?

Reply:
Sorry, i do not understand the question.
Right now, this module returns `entity.to_json_dict()`, which is what the nailgun thinks to be the API.